src/main/java/io/lighty/yang/validator/formats/Format.java

```diff
@@ -40,18 +41,24 @@ public void createOptions(final LyvParameters lyvParameters) {
     }
 
     @Override
-    public void init(final Configuration config, final EffectiveModelContext context, final Module module,
+    public void init(final Configuration config, final EffectiveModelContext context,
             final SchemaTree schemaTree) {
         final String format = config.getFormat();
         for (final FormatPlugin plugin : this.formatPlugins) {
             if (plugin.getHelp().getName().equals(format)) {
                 this.usedFormat = plugin;
-                this.usedFormat.init(context, module, schemaTree, config);
+                this.usedFormat.init(context, schemaTree, config);
             }
         }
     }
 
-    public void emit() {
-        this.usedFormat.emitFormat();
+    public void emit(final Module module, final LyvEffectiveModelContext lyvcontext, final SchemaTree schemaTree,
+            final Configuration config) {
+        this.usedFormat.testedModule = module;
```

Comment:
Why do we need `this.usedFormat.testedModule = module;`?

Reply:
Removed this.testedModule